index.js

```diff
@@ -398,7 +401,7 @@ async function run() {
     const forceNewDeployInput = core.getInput('force-new-deployment', { required: false }) || 'false';
     const forceNewDeployment = forceNewDeployInput.toLowerCase() === 'true';
     const desiredCount = parseInt((core.getInput('desired-count', {required: false})));
-    const enableECSManagedTagsInput = core.getInput('enable-ecs-managed-tags', { required: false }) || 'false';
+    const enableECSManagedTagsInput = core.getInput('enable-ecs-managed-tags', { required: false }) || '';
     const enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
```

Comment:
This is still being used in `runTask` - can we include the changes for the enable ecs managed tags behavior in `runTask` as well to keep the behavior consistent?

Reply:
In fact we can just move
```
  enableECSManagedTags = ""
  if (enableECSManagedTagsInput !== "") {
   enableECSManagedTags = enableECSManagedTagsInput.toLowerCase() === 'true';
  }
```
here and set the param in `updateEcsService` and `runTask` only if it's not set to ""

Reply:
when enableECSManagedTags = “”, ecs.updateService(params) get an error "SerializationException: Unexpected token received from parser"
so enableECSManagedTags is set to null by default

I actually checked with the following code in my aws

```
const ecs = new ECS();
  
let params = {
    cluster: "cluster-name",
    service: "servicepname",
    taskDefinition: "arn:aws:ecs:us-east-1:000000000:task-definition/some-task:1",
    enableECSManagedTags: "", // parse error
    // enableECSManagedTags: null, // ok
    desiredCount: 1,
};

await ecs.updateService(params);
```

Reply:
Great find!